public/src/cards.js

```diff
@@ -95,6 +95,8 @@ let events = {
       pack[name] = 1
     }
     App.update()
+    if (document.hidden && App.state.beep)
```

Comment:
We can just do away with the document checking. If they have beep on pack enabled, beep it!